src/test/java/org/springframework/data/couchbase/repository/query/PartTreeN1qBasedQueryTest.java

```diff
@@ -117,12 +126,129 @@ public void testGetCountExcludesDynamicSortClause() throws Exception {
 
 	}
 
+	@Test
```

Comment:
Thanks for pointing out SpelAwareProxyProjectionFactory. The mocking is bit less now, I think we should move it to integration test next release?